src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/web/ReportTests.kt

```diff
@@ -53,6 +54,29 @@ class ReportTests : IntegrationTest()
         assertThat(pinnedReports[0].name).isEqualTo("html")
     }
 
+    @Test
+    fun `only report reviewers can get report drafts`()
+    {
+        val url = "/report-drafts/"
+        assertWebUrlSecured(url,
+                setOf(ReifiedPermission("reports.review", Scope.Global())),
+                contentType = ContentTypes.json)
+    }
+
+    @Test
+    fun `report reviewers can get report drafts`()
+    {
+        val url = "/report-drafts/"
+        val response = webRequestHelper.loginWithMontaguAndMakeRequest(url,
+                setOf(ReifiedPermission("reports.review", Scope.Global())),
+                contentType = ContentTypes.json)
+
+        val data = JSONValidator.getData(response.text)
```

Comment:
I didn't add a schema because I think we don't normally do that for internal (web) endpoints?